index.html

```diff
@@ -1336,6 +1336,15 @@ <h5>
           <div class="note">
             Other specifications may expand on this definition.
           </div>
+          <div class="note">
+            <p>
+              <i>This note is not normative.</i>
+            </p>
+            When an <code>"insertFromPaste"</code> <code>beforeinput</code>
```

Comment:
```suggestion
            When an <code>"insertFromPaste"</code> {{beforeinput}}
```

I think insertFromPaste should also be defined using `dfn` to allow linking, but out of scope here.